### ISSUE TITLE
skin/ct/t problem

### ISSUE THREAD

**Issue by Smofly** [Nerus87/Team-Balance#2]
Sometimes a ct spawns at t spawn and a t spawn at ct spawns. after it have move around people

**Comment by Nerus87** [comment]
Yes, I had this problem in swap teams plugin, caused by pre round spawn. CSS don't have mechanism before spawn player like TF2 or CSgo. In this case need to change players 0.1 sec before round start.
